Arrowgene.Ddon.GameServer/Handler/LobbyLobbyDataMsgHandler.cs

```diff
@@ -53,9 +54,12 @@ public override void Handle(GameClient client, StructurePacket<C2SLobbyLobbyData
             else
             {
                 // We are in one of the common areas where players can see eachother
+		        List<GameClient> otherClients = Server.ClientLookup.GetAll()
+                                                    .Where(x => x != null)
+                                                    .ToList();
                 foreach (GameClient otherClient in Server.ClientLookup.GetAll())
                 {
-                    if (otherClient == client)
+                    if (otherClient == null || otherClient == client)
```

Comment:
We still want to check and make sure client and otherClient are not the same. otherwise the client will get backets for itself.

Reply:
It's already doing that, isnt it

Reply:
well it still is checking that, im confused

Reply:
You are right, I thought it was just checking both clients for `null`.